x-pack/solutions/security/plugins/security_solution/public/siem_migrations/dashboards/components/dashboard_table/index.tsx

```diff
@@ -0,0 +1,196 @@
+  EuiBasicTable,
+} from '@elastic/eui';
+import React, { useCallback, useMemo, useState } from 'react';
+
+import type { SiemMigrationFilters } from '../../../../../common/siem_migrations/types';
+import type { DashboardMigrationDashboard } from '../../../../../common/siem_migrations/model/dashboard_migration.gen';
+import { useMigrationDashboardsTableColumns } from '../../hooks/use_migration_dashboards_table_columns';
+import { useGetMigrationDashboards } from '../../logic/use_get_migration_dashboards';
+import { MigrationTranslationResult } from '../../../../../common/siem_migrations/constants';
+import * as i18n from './translations';
+import type { DashboardMigrationStats } from '../../types';
+import { MigrationDashboardsFilter } from './filters';
+import { convertFilterOptions } from './utils/filters';
+import { EmptyMigration, SearchField } from '../../../common/components';
+import type { FilterOptionsBase } from '../../../common/types';
+
+const DEFAULT_PAGE_SIZE = 10;
+const DEFAULT_SORT_FIELD = 'translation_result';
+const DEFAULT_SORT_DIRECTION = 'desc';
+
+export interface MigrationDashboardsTableProps {
+  /**
+   * Re-fetches latest dashboards migration data
+   */
+  refetchData?: () => void;
+
+  /**
+   * Migration stats
+   */
+  migrationStats: DashboardMigrationStats;
+}
+
+/**
+ * Table Component for displaying SIEM dashboards migrations
+ */
+export const MigrationDashboardsTable: React.FC<MigrationDashboardsTableProps> = React.memo(
+  ({ refetchData, migrationStats }) => {
+    const migrationId = migrationStats.id;
+
+    const [pageIndex, setPageIndex] = useState(0);
+    const [pageSize, setPageSize] = useState(DEFAULT_PAGE_SIZE);
+    const [sortField, setSortField] =
+      useState<keyof DashboardMigrationDashboard>(DEFAULT_SORT_FIELD);
+    const [sortDirection, setSortDirection] = useState<'asc' | 'desc'>(DEFAULT_SORT_DIRECTION);
+    const [searchTerm, setSearchTerm] = useState<string | undefined>();
+
+    // Filters
+    const [filterOptions, setFilterOptions] = useState<FilterOptionsBase | undefined>();
+
+    const filters = useMemo<SiemMigrationFilters>(
+      () => ({ searchTerm, ...convertFilterOptions(filterOptions) }),
+      [searchTerm, filterOptions]
+    );
+
+    const {
+      data: { migrationDashboards, total } = { migrationDashboards: [], total: 0 },
+      isLoading: isDataLoading,
+    } = useGetMigrationDashboards({
+      migrationId,
+      page: pageIndex,
+      perPage: pageSize,
+      sortField,
+      sortDirection,
+      filters,
+    });
+
+    const [selectedMigrationDashboards, setSelectedMigrationDashboards] = useState<
+      DashboardMigrationDashboard[]
+    >([]);
+    const tableSelection: EuiTableSelectionType<DashboardMigrationDashboard> = useMemo(
+      () => ({
+        selectable: (item: DashboardMigrationDashboard) => {
+          return (
+            !item.elastic_dashboard?.id &&
+            item.translation_result === MigrationTranslationResult.FULL
```

Comment:
```suggestion
            item.translation_result !== MigrationTranslationResult.FAILED
```

I think only `FAILED` and `Installed` should be unselectable.

Because users should be able to install `Partially Translated` dashboards.

Reply:
Actually, it looks like it should be

```
(
  !item.elastic_dashboard?.id &&
  item.status !== SiemMigrationStatus.FAILED &&
  item.translation_result !== MigrationTranslationResult.UNTRANSLATABLE
)
```

Reply:
Done in https://github.com/elastic/kibana/pull/234486/commits/3d7b2946e405c4cfeba813d6e972437731e43cab